src/server/verification/faceRecognition/human.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { humanApi, kairosAPI } from 'express-kairos-faceverification'
+import conf from '../../server.config'
+
```

Comment:
you can destructurize
```
const { id: app_id, key: app_key } = conf.kairos
```
then just
```
const apiKairos = new kairosAPI({ app_id, app_key }, 'test', true)
```

Also you can omit defining humanInstance constant as it just exports and dont uses anywhere else. Just:
`export default new humanApi( ....`

Reply:
done